necroHeal.lic

```diff
@@ -0,0 +1,171 @@
+=begin
+  Necromancer healing script
+  Run to area defined in YAML and kill creatures until fully healed
+  Uses combat trainer and YAML file
+  Consume logic via combat trainer
+  v.01 by Kalros
+
+TO DO:
+  - Need to see if there is a graceful exit for combat trainer
+    potential for items to get left behind.
+
+=end
+
+## USER VARIABLES - Set
+
+## Uses base-hunting.Yaml
+$huntZone = 'goblins'
+
+## Set to your Name-heal.yaml file.
+$yamlFile = 'heal'
+
+if $huntZone == 'XXX'
+  echo "You must open the script and set your hunting zone and yamlFile"
+  exit
+end

```

Comment:
We should use the yaml files for these settings inside the class instead of global variables in the script.